README.md

```diff
@@ -1 +1,8 @@
-# Hello-world
\ No newline at end of file
+# Microverse first project
+The project is simple and is intended to test my git and github technical skills
+## Built with
+  Html
+  Css
+  ## Author
+  GitHub:@torobucii
+  
```

Comment:
- [ ] Kindly I recommend you to update your README file with the new [README template](https://github.com/microverseinc/readme-template)  , **All main headers** from the [README template](https://github.com/microverseinc/readme-template) should be present in a yours repo and Please Make sure the you has added the **section Future Features**, including 1-3 potential features to be added..

Reply:
Fixed.